frontend/src/components/wrappers/Input.jsx

```diff
@@ -17,6 +17,11 @@ export default function Input({ children, label }) {
 }
 
 Input.propTypes = {
-  children: PropTypes.node.isRequired,
-  label: PropTypes.string.isRequired,
+  children: PropTypes.node,
+  label: PropTypes.string,
+};
+
+Input.defaultProps = {
+  children: null,
+  label: null,
```

Comment:
```suggestion
  children: undefined,
  label: undefined,
```
here, I think children should be required since we're wrapping, but for the scope of the PR this works great; I'll take note of updating Input to be a standalone component